### DISCUSSION TITLE
Audio via Bluetooth Esp32-C3

### DISCUSSION THREAD

**Discussion by nemoscaos** [pschatzmann/ESP32-A2DP#324] [Q&A] [answered]
Hello everybody
I've seen many similar threads, but none that answer my problem in full.

my old project has Esp32 with bluetooth and I send audio, all ok, all perfect.
in my new project i have to use bluetooth 5.0 with esp32-c3
but I understood that it is not supported because it is BLE only.

So I don't understand which roads I can take to have a bluetooth that plays an audio file in i2s

have you tried BLE?

can you help me ???
thank you
Steve

**Comment by pschatzmann** [comment]
Just use a regular ESP32 if you need Bluetooth Audio!

**Comment by nemoscaos** [comment]
Hi Phil, I've been following you for a while...nice to meet you.

it's true!!! I would never change...
but in today's world a project with bluetooth 4.2 collides with bluetooth 5.0 projects and is always the loser even if it works very well!!
so i have to make it work with bluetooth 5.0 somehow...

**Comment by pschatzmann** [comment]
I suggest that you just check with Espressif what their plans are...
I would be surprised if they are not working on this.

**Comment by nemoscaos** [comment]
I found something on the espressif site...can I try to install micropython and activate bluetooth...could this be the right way? I'm not an expert, I'm experimenting...

**Reply by pschatzmann** [reply]
I thought you wanted audio!

**Reply by nemoscaos** [reply]
yes I want audio I'm just trying to figure out how to move ... I'm not very experienced, I have to have 2 things: bluetooth and audio! I'm still studying today...

**Comment by pschatzmann** [comment, marked as answer]
Since the ESP32 (software and most likely hardware) does not support this you might need to use a separate hardware module
